app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name), id: 'login-btn' %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_registration_path(resource_name), id: 'sign-up-btn' %><br />
+<% end %>
```

Comment:
Hello @AnselemOdims thank you for your feedback, for signup page the button to register is the one that says `Next` at the top right corner and the button to login is the button called `login` which is located at the top right corner as well I tried, but so that others won't get confused I will add a text indicating the purpose of the buttons at the bottom